validator/test/fixtures/gke_restrict_client_auth_methods/restrict_gke_client_auth_methods/constraints/data.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
```

Comment:
the file path here doesn't seem to follow the convention? shouldn't it be `.../gke_restrict_client_auth_methods/constraints/restrict_gke_client_auth_methods/data.yaml` instead?

Reply:
updated